devices/Hx711/Hx711.Sample/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
```

Comment:
Maybe improve this file and copy/paste from other bindings where it's a bit more clean?